design/ceph/object/store.md

```diff
@@ -70,6 +72,11 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  radosNamespaces:
+    # If true, create RADOS namespaces for this object store.
```

Comment:
```suggestion
    # If true, create RADOS namespaces for this object store.
```
the namespace for RADOS will created automatically or user needs to create this?

Reply:
There is no namespace per se, this is more like annotations in the pool, so that RGW data will be stored in that location